### ISSUE TITLE
Unable to compile with SDK 3.8 on cloudpebble.net

### ISSUE THREAD

**Issue by clach04** [openhab/openhab.pebble#4]
This is due to an issue with cloudpebble.net. Workaround, add the following lines to appinfo.json:

```
"sdkVersion": "3",
"targetPlatforms": [
    "aplite",
    "basalt"
].
```

**Comment by clach04** [comment]
Potentially also resolved by issue #6

**Comment by llamahunter** [comment]
fixed by 26f6bb182a3519f78173d823d713abe74d85b94e
